.github/workflows/helm.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v2
       - name: Get Helm
         run: |
-          curl -Lo ./helm.tar.gz https://get.helm.sh/helm-v2.17.0-linux-amd64.tar.gz
+          curl -Lo ./helm.tar.gz https://get.helm.sh/helm-v3.5.2-linux-amd64.tar.gz
```

Comment:
Here you checkout@v2 but you reference `v3.5.2`.

Reply:
those are different steps. `actions/checkout@v2` is the action that checks out the code, the action's version is `v2`.
And in this line I download `v3.5.2` version of helm. Those steps are not connected.